charts/github-actions-runners/templates/horizontal-runner-autoscaler.yaml

```diff
@@ -0,0 +1,19 @@
+apiVersion: actions.summerwind.dev/v1alpha1
+kind: HorizontalRunnerAutoscaler
+metadata:
+  name: runnerautoscaler
```

Comment:
Maybe you could use the name defined in the _helpers.tpl to provide some entropy?

Reply:
Using now